loader.c

```diff
@@ -293,13 +304,29 @@ dloader_p api_load(const char *filename)
     ElfW_Reloc *relocs =
         (ElfW_Reloc *)(load_bias + get_dynamic_entry(dynamic, ELFW_DT_RELW));
     size_t relocs_size = get_dynamic_entry(dynamic, ELFW_DT_RELWSZ);
+
+    /*
+     * FIXME
+     * There is no RELA in ARM instead of REL,
+     * someone should make the condition code better.
```

Comment:
Skip the line starting with "someone". `FIXME` section only denotes further modifications.